packages/graphql/tests/e2e/deprecated/delete-relationship-via-delete-additional-labels.e2e.test.ts

```diff
@@ -104,7 +104,13 @@ describe("Delete Subscriptions when only nodes are targeted - when nodes employ
                 subscriptions: new TestSubscriptionsPlugin(),
             },
         });
-        server = new ApolloTestServer(neoSchema);
+        // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Why is this disable required?

Reply:
The Apollo context function is typed to always be async... But the contents don't always await anything. So ESLint complains. 🙈